apps/camera_pipe/camera_pipe.cpp

```diff
@@ -333,13 +332,10 @@ int main(int argc, char **argv) {
         .bound(tx, 0, (out_width/32)*32)
```

Comment:
We generate better code if the height and width are a multiple of the tile size. Isn't that what this is trying to do? If so, then even this depends on the schedule which in turn depends on the target.